Lib/test/test_audit.py

```diff
@@ -118,5 +118,19 @@ def test_socket(self):
         self.assertEqual(events[2][0], "socket.bind")
         self.assertTrue(events[2][2].endswith("('127.0.0.1', 8080)"))
 
+    def test_gc(self):
+        import_helper.import_module("gc")
```

Comment:
The gc module must always be available, so I don't think that import_module() is useful to skip the test if it's missing, no?